test/integration/scheduler/scoring/priorities_test.go

```diff
@@ -296,18 +296,18 @@ func TestNodeResourcesScoring(t *testing.T) {
 // works correctly.
 func TestNodeAffinityScoring(t *testing.T) {
 	testCtx := initTestSchedulerForScoringTests(t, nodeaffinity.Name, nodeaffinity.Name)
-	// Add a few nodes.
-	_, err := createAndWaitForNodesInCache(testCtx, "testnode", st.MakeNode(), 4)
-	if err != nil {
-		t.Fatal(err)
-	}
 	// Add a label to one of the nodes.
 	labelKey := "kubernetes.io/node-topologyKey"
 	labelValue := "topologyvalue"
 	labeledNode, err := createNode(testCtx.ClientSet, st.MakeNode().Name("testnode-4").Label(labelKey, labelValue).Obj())
```

Comment:
You should also wait for this `labeledNode` to be presented in the cache, no?

Reply:
Definietely. I've made sure of it.